### PR TITLE
this is merge request test !

### PR DESCRIPTION
**Pull request by jollyja**
pls merge, 3Q.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
ok, well done.

**Comment by ghost** [comment]
ok

File: pom.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
 
   <name>demo</name>
   <url>http://maven.apache.org</url>
+  <description>this is just a merge request test</description>
```
Comment (ghost) [inline pom.xml:12]: good

**Review by ghost — APPROVED** [review]
done

**Review by ghost — CHANGES_REQUESTED** [review]
done done

**Review by ghost — COMMENTED** [review]
xx

**Review by ghost — APPROVED** [review]
approve

**Comment by jollyja** [comment]
close

**Pull request merged**